### ISSUE TITLE
RichTextLabel renders text as faint block - new and old projects affected

### ISSUE THREAD

**Issue by TheOptimizer** [godotengine/godot#64395]
### Godot version

Godot Engine v3.5.stable.official.991bb6ac7

### System information

Windows 10, OpenGL ES 2.0 Renderer: Intel(R) HD Graphics 630

### Issue description

3 days ago, I mocked up some UI elements and entered some placeholder text into my RichTextLabel via the Node Inspector. The default godot font was used. It displayed my text just fine. I saved my work and closed the editor. 

When I reopened my project file just now, all text have failed to render properly. They show up as faint blocks instead. I attempted to reproduce the bug in a new project and was able to do so. The faint blocks looked like this:

![image](https://user-images.githubusercontent.com/95207599/184541660-7f222cf0-394a-48bf-85c0-9ebe84a372c1.png)

The only thing of note that happened between the time it last worked and now are the two Windows update that I installed KB5012170 and KB5016616. No error messages are seen anywhere.

### Steps to reproduce

1. Open up a new project, save a Node2D as a placeholder scene. 
2. Add RichTextLabel as a child node and add some placeholder text via the node inspector. 
3. Observe that the text fail to render properly and show up as faint blocks. (See the minimum reproduction project below)

### Minimal reproduction project

[RichTextTest.zip](https://github.com/godotengine/godot/files/9333555/RichTextTest.zip)

**Comment by Calinou** [comment]
I can't reproduce this on 3.5.stable on Linux (NVIDIA 515.57):

![image](https://user-images.githubusercontent.com/180032/184547408-3536d7d6-cddd-4223-a1d7-93ea09093578.png)

Can you reproduce this after disabling batching for both project and editor in the Project Settings?

**Comment by TheOptimizer** [comment]
Whoa! Changing GLES2 to GLES3 worked for both projects!! The text now shows up again. Thank you so much for the helpful suggestion @Calinou 

Though, I'm still left wondering why it happened - I chose GLES2 assuming it is best for hardware compatibility in the first place.

**Comment by Calinou** [comment]
Can you reproduce the issue after switching back to GLES2 and using a DynamicFont (.ttf/.otf/.woff/.woff2)?

**Comment by TheOptimizer** [comment]
The behaviour I experienced was very strange. So pay attention to the following sequence of events:

1) I opened my original project and switched back from GLES3 to GLES2.  It did cause the issue to reappear.
2) I added a new RichTextLabel at the corner and as expected it rendered text as blocks.
3) I imported Lexend.ttf from Google, used it on the new label, run the project, and suddenly all text rendered normally again - including the original text in the default font.
4) Most curiously, the minimum reproduction project now also renders text properly after switching back to GLES2 without needing to import Lexend.

Not sure what this does to your mipmap theory - I never messed with any settings related to that. I have the latest Intel GPU drivers.
